demo/package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "build:wasm-st": "wasm-pack build --target web",
-    "build:wasm-mt": "cross-env RUSTFLAGS='-C target-feature=+atomics,+bulk-memory' rustup run nightly-2021-02-11 wasm-pack build --target web --out-dir pkg-parallel -- --features parallel -Z build-std=panic_abort,std",
+    "build:wasm-mt": "cross-env RUSTFLAGS='-C target-feature=+atomics,+bulk-memory' rustup run nightly-2021-07-29 wasm-pack build --target web --out-dir pkg-parallel -- --features parallel -Z build-std=panic_abort,std",
```

Comment:
Looks like it's due to missed `mutable-globals` feature here.

Reply:
Let's see... https://github.com/GoogleChromeLabs/wasm-bindgen-rayon/commit/24b9456bf7a3b6f14b6505d8c66f2c457d69ef79

Reply:
Yeah that was it.